src/main/views/steps-to-making-your-claim.njk

```diff
@@ -7,19 +7,42 @@
 {% endblock %}
 
 {% block content %}
-  <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">{{ h1 }}</h1>
-  <h2 class="govuk-heading-l">{{ section1.title }}</h2>
-  <hr class="govuk-section-break govuk-section-break--visible">
-  <a href="{{ PageUrls.DOB_DETAILS }}" class="govuk-link">{{ section1.link1Txt }}</a>
-  <hr class="govuk-section-break govuk-section-break--visible">
-  <a href="{{ PageUrls.ADDRESS_DETAILS }}" class="govuk-link">{{ section1.link2Txt }}</a>
-  <hr class="govuk-section-break govuk-section-break--visible">
-  <a href="{{ PageUrls.UPDATE_PREFERENCES }}" class="govuk-link">{{ section1.link3Txt }}</a>
-  <hr class="govuk-section-break govuk-section-break--visible">
+
+  <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">{{h1}} TODO</h1>
+
+  {% for section in sections %}
+  
+  <table class="govuk-table">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row ">
+        <th scope="col" class="govuk-table__header govuk-grid-column-three-quarters">{{ section.title }}</th>
+      </tr>
+    </thead>
+    {% for link in section.links %}
+
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td scope="row" class="govuk-table__cell govuk-grid-column-one-half">  
+        {% if link.url !="" %}
+        <a href= "{{ link.url }}" class="govuk-link">
+        {{ link.linkTxt }}
+        </a>
+        {% else %}
+        {{link.linkTxt}}
+        {% endif %}
+        </td>
+        <td scope="row" class="govuk-table__cell govuk-grid-column-one-quarter">TODO</td>
```

Comment:
I think the 'TODO' etc text are meant to be 'Tags' (https://design-system.service.gov.uk/components/tag/) from looking at the design. https://c3ibml.axshare.com/#id=j2fzbj&p=task_list_1__steps_to_making_your_claim&g=1

Reply:
Tags for statuses on the right hand side of the grid don't need to be implemented for this user story. Good spot Angela, just to say we will implement them in a follow up story one we know what each tag state can be from UCD.